app/models/message.rb

```diff
@@ -80,9 +80,7 @@ def update_client_store
   def send_update(object, user, state, channel)
     self.state_value = state
     object = Api::V1::MessageSerializer.new(object, {exclude:Message.reflections.keys.map(&:to_sym)})
-    PushService
-      .new(channel: channel, event: 'update_store', data: {item:object, sender:user, operation: :create})
-      .notify unless channel.empty?
+    PushService.new.send_update_store(channel, {item:object, sender:user, operation: :create}) unless channel.empty?
```

Comment:
Line is too long. [116/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.